src/helpers/applicationUpdates.spec.ts

```diff
@@ -90,12 +90,12 @@ describe('findAndInstallServiceWorkerUpdate', () => {
     const update = jest.fn();
     const sw = {update: update as ServiceWorkerRegistration['update']} as ServiceWorkerRegistration;
 
-    let finishUpdate: () => void = () => null;
+    let finishUpdate: (value: any) => void = () => null;
```

Comment:
I've looked again on this
We can do `let finishUpdate: () => void = () => null;`
and then instead of
```
update.mockReturnValue(new Promise((resolve) => finishUpdate = resolve));
```
we have to do:
```
update.mockReturnValue(new Promise<void>((resolve) => finishUpdate = resolve));
```